### PR TITLE
[SHEPHERD] Fixed scoreboard scaling

### PR DESCRIPTION
**Pull request by charleshong3**
Latest changes to scoreboard that fix scaling

### CODE REVIEW

File: shepherd/static/scoreboard.js [hunk 1]
```diff
@@ -272,93 +270,98 @@ document.addEventListener("DOMContentLoaded", function(event) {
...
+        // }
+        // ctx_bottom.textAlign = "left";
+        // ctx_bottom.fillStyle = "white";
+        // if(master_gold_score < 10){
+        //     ctx_bottom.fillText(master_gold_score.toString(), width - 527, 95);
+        // } else if (master_gold_score < 100){
+        //     ctx_bottom.fillText(master_gold_score.toString(), width - 543, 95);
+        // } else {
+        //     ctx_bottom.fillText(master_gold_score.toString(), width - 558, 95);
+        // }
```
Comment (JHGZhou) [inline shepherd/static/scoreboard.js]: delete large chunks of commented out code

File: shepherd/static/scoreboard.js [hunk 2]
```diff
@@ -89,8 +87,8 @@ document.addEventListener("DOMContentLoaded", function(event) {
 
         bottom = document.getElementById("bottom_bar");
         ctx_bottom = bottom.getContext("2d");
-        multipliercanvas = document.getElementById("multipliers");
-        ctx_multiplier = multipliercanvas.getContext("2d");
+        // multipliercanvas = document.getElementById("multipliers");
+        // ctx_multiplier = multipliercanvas.getContext("2d");
```
Comment (JHGZhou) [inline shepherd/static/scoreboard.js]: delete commented out code

**Review by JHGZhou — DISMISSED** [review]
the scoreboard looked good when you showed it to me, just clean up the two commented out chunks

**Review by brandonxxlee — DISMISSED** [review]
lgtm, as long as it works

**Review by brandonxxlee — APPROVED** [review]

**Pull request merged**